articles/openstack/ostack-faq.md

```diff
@@ -229,7 +229,9 @@ The answer to this depends on which of UKCloud's OpenStack services you are usin
 | Region              | OpenStack version                      | LBaaS supported? |
 |---------------------|----------------------------------------|------------------|
 | COR00005 <BR> FRN00006 | OpenStack Platform 10 <BR>(Newton Release) | No, as Red Hat's Newton release does not support native Key Management as a Service (KMaaS) to underpin encryption  |
-| COR00005-2 | OpenStack Platform 13 <BR> (Queens Release) | Yes |
+| COR00005-2 | OpenStack Platform 13 <BR> (Queens Release) | In Beta* |
```

Comment:
Just noticed that the column header says LBaaS supported - assume this is a copy and paste error. What should this be? D@RE supported?